helloworld/helloworld/ViewController.swift

```diff
@@ -17,5 +17,7 @@ class ViewController: UIViewController {
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/marianoabdala/HelloWorld/blob/22929daac6424ddce1b82dc0f5f5b5822ee363b7/helloworld/helloworld/ViewController.swift#L20"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>